### PR TITLE
Refactors code to match es2015 modules

### PR DESCRIPTION
**Pull request by vicnicius**

### CODE REVIEW

**Comment by vicnicius** [comment]
This PR is an extension of the work on https://github.com/catarse/catarse.js/pull/185.

By explicitly applying modularization through out the code, it became clear that mithril-postgrest provides functionalities completely independent from mithril itself, and I think we should go on this way with the project. 

The changes here would be a first step on making mithril.postgrest independent from mithril. It removes it from the mithril namespace and exposes it as both an IIFE and an UMD js module (named postgrest), using rollup to build the bundles.

The next steps would be to use a proper request abstraction (perhaps the XMLHttpRequest api itself, or a more universal approach) and rethink naming aspects, to remove the cognitive relation to mithril completely.

This move aims to make it possible to any web app to easily integrate with a postgrest back-end. 

@diogob @devton @adrianob I would love to hear what you think.

**Comment by diogob** [comment]
I like the idea. We can just copy mithril code for `request` and `prop`. Since both are used and have a good interface. Also should be a very small portion of code, so having a duplicate would not be an issue.

File: karma.conf.js [hunk 1]
```diff
@@ -33,21 +32,21 @@ module.exports = function(config) {
...
-        sourceMap: 'inline'
-      },
-      filename: function(file) {
-        return file.originalPath.replace(/\.js$/, '.es5.js');
-      },
-      sourceFileName: function(file) {
-        return file.originalPath;
-      }
-    },
+    // preprocessors: {
```
Comment (diogob) [inline karma.conf.js]: Let's remove the commented out code ;)

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import babel from 'rollup-plugin-babel';
```
Comment (diogob) [inline rollup.config.js]: babel is such an appropriate name for this project, in two years from now nobody will be able to understand each other :P

Comment (vicnicius) [inline rollup.config.js]: I'm counting the days to drop it! x)

**Pull request merged**